README.md

```diff
@@ -60,6 +104,11 @@ The job UUID or name to download the artifact from.
 
 The build UUID to download the artifact from.
 
+### Relocation
+
+If a file needs to be renamed or moved before upload or after download, a special pattern is supported: `source -> destination`.
```

Comment:
Sounds like this might not be true anymore?

Reply:
good catch. i'll fix the readme